src/Environments/Windows/MsPrintfFormatParser.cs

```diff
@@ -47,7 +47,9 @@ protected override DataType MakeDataType(PrintfSize size, char cDomain)
         {
             if (cDomain == 'S')
             {
-                return program.TypeFactory.CreatePointer(PrimitiveType.WChar, base.pointerSize);
+                return program.TypeFactory.CreatePointer(
```

Comment:
Consider making CreatePointer bit-sized as well -- we should be in general be moving towards bit orientation when working with primitive types. `Pointer` is basically a `PrimitiveType` of domain `Pointer` with extra type information. Then we don't have to do all these divisions.